src/Protocol/Protocol_1_11.cpp

```diff
@@ -388,7 +388,7 @@ void cProtocol_1_11_0::SendSpawnMob(const cMonster & a_Mob)
 	// TODO: Bad way to write a UUID, and it's not a true UUID, but this is functional for now.
 	Pkt.WriteBEUInt64(0);
 	Pkt.WriteBEUInt64(a_Mob.GetUniqueID());
-	Pkt.WriteVarInt32(static_cast<UInt32>(a_Mob.GetMobType()));
```

Comment:
Legacy sounds like the wrong name for this function - we don't want to have 'Legacy' code in every protocol including the latest one. Maybe `GetProtocolMobID` or something instead?